tests/queries/0_stateless/02377_optimize_sorting_by_input_stream_properties_explain.reference

```diff
@@ -89,10 +148,25 @@ Sorting (Sorting for ORDER BY)
 Sorting (Global): plus(a, 1) ASC
 Sorting (Chunk): a ASC
 Sorting (Chunk): a ASC
+-- QUERY (analyzer): set optimize_read_in_order=1;set max_threads=3;set query_plan_remove_redundant_sorting=0;EXPLAIN PLAN actions=1, header=1, sorting=1 SELECT a, z FROM (SELECT sipHash64(a) AS a, a + 1 AS z FROM (SELECT a FROM optimize_sorting ORDER BY a + 1)) ORDER BY a + 1
+Sorting (None)
+Sorting (Sorting for ORDER BY)
+Sorting (Global): plus(a_0, 1_UInt8) ASC
+Sorting (Global): plus(default.optimize_sorting.a_3, 1_UInt8) ASC
```

Comment:
is it ok that they're different?

Reply:
Yes, `Sorting (Global): plus(a_0, 1_UInt8) ASC` is `ORDER BY` output and `plus(a_0, 1_UInt8) UInt64` is the sorting column